Decisions/scs-0310-v1-publishing_method_for_metering_data.md

```diff
@@ -0,0 +1,48 @@
+---
+title: pushing as publishing method for metering
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+
+# Introduction
+
+In the past we notice missing events in the telemetry stack of openstack. this results in situations where the Service provider may think that a resource is still in use while the owner shut it down.
+Such missing data makes it difficult for the CSP to provide accurate invoices
+
+# Motivation
+
+we want to fill the gab between measuring resource consumption and having reliable data for invoicing.
```

Comment:
Sentences start with a capital letter. 'We[...]

Reply:
also: `gap` not `gab` :wink: